sdk/monitor/opentelemetry-exporter/samples/@azure/storage-blob/README.md

```diff
@@ -0,0 +1,71 @@
+# Getting started with samples
+
+Code cloned from [azure-sdk-for-js/storage-blob/samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-blob/samples).
+
+## `@azure/monitor-opentelemetry-exporter`
+
+All changes required to enable exporting trace data to Azure Monitor were made in `tracing.ts` / `tracing.js`.
+
+In order for auto-instrumentation to take effect, `tracing` must be imported **before** you `import` or `require` any other module into your application!.
+
+![AzureMonitorImage](./images/storage.png)
+
+## Get account credentials of your storage account
+
+- In the [Azure Portal](https://portal.azure.com), go to **Dashboard > Storage > _your-storage-account_**.
+- Note down the "AccountName", "AccountKey" obtained at **Access keys** and "AccountSAS" from **Shared access signature** under **Settings** tab.
+  Before running any of the samples, update with the credentials you have noted down above.
+
+### Authenticating with Azure Active Directory
+
+If you have [registered an application](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app) with an Azure Active Directory tenant, you can [assign it to an RBAC role](https://docs.microsoft.com/en-us/azure/storage/common/storage-auth-aad) in your Azure Storage account. This enables you to use the Azure.Identity library to authenticate with Azure Storage as shown in the [azureAdAuth.ts sample](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/storage/storage-blob/samples/typescript/src/azureAdAuth.ts).
+
+## JavaScript sample
+
+Copy the sample to your samples folder and use `node` to run it.
+
+```bash
+node sample.js
+```
+
+## TypeScript sample
+
+If you don't have Typescript installed, then use `npm` to install it first.
+
+```bash
+npm install -g typescript
+```
+
+One way to run Typescript samples is to use `ts-node`. To install `ts-node`, run the below in your sample folder
+
+```bash
+npm install -g ts-node
+```
+
+Copy the sample to your samples folder and use `ts-node` to run it.
+
+```bash
+ts-node sample.ts
+```
+
+## Browser sample
```

Comment:
Could probably link to our centralized bundling docs as well (https://github.com/Azure/azure-sdk-for-js/blob/master/documentation/Bundling.md)

Reply:
Come to think of it, I should remove the browser section since the OTel exporter won't work in the browser for now. Maybe once I connect it to `core-http`, I will get that for free? 🤞

Reply:
Might have to tweak some things, like not calling "unref" on the result of `setTimeout` since that's not a thing in the browser.